projects/transformers/README.md

```diff
@@ -10,29 +10,51 @@ Other accepted ways are passing arguments directly in command line or through a
 
 ## How to run - multiple nodes
 
-To run it in a cluster, first define the location of your cluster yaml file in `RAY_CONFIG_FILE` and the location of your AWS certification file (.pem file) in `AWS_CERT_FILE`:
+To run it in multiple nodes, you will require the run script from transformers-cli-utils. It can be found in nta/NumentaCorp/infrastructure/ray/transformers-cli-utils. Make sure to add it to the PATH after downloading.
+
+`export PATH=$PATH:~/nta/infrastructure/ray/transformers-cli-utils`
+
```

Comment:
Please remove these lines. See slack comments

Reply:
I removed the full path. I do need to make some mention of where to find the tool for our internal collaborators, otherwise the readme is useless. 

The alternative is to move transformers-cli-utils into this folder. I am not opposed to that. But I think that will be a higher security risk than just a quick mention that the tools exists under a given private repository.

Reply:
@lscheinkman removed the references of where to find transformers-cli-utils. I will add this to a separate PR so this discussion doesn't block merging the other changes. .